internal/experimental/experimental.go

```diff
@@ -0,0 +1,75 @@
+package experimental
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	envName = "NOTATION_EXPERIMENTAL"
+	enabled = "1"
+)
+
+// IsDisabled determines whether an experimental feature is disabled.
+func IsDisabled() bool {
+	return os.Getenv(envName) != enabled
```

Comment:
should we trim the white spaces here, for both key and value?

Reply:
For what purpose do we need to trim key and value?

Reply:
In scenario where user calls following command to set env variable.
`export NOTATION_EXPERIMENTAL=" 1"`

Its an edge case, I am fine to ignore this.